manageiq-messaging.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'activesupport', '~> 5.2.4.3'
+  spec.add_dependency 'activesupport', '~> 5.2.6'
```

Comment:
Should this be loosened to `~> 5.2` or does something less than 6 break it?

Reply:
I was matching https://github.com/ManageIQ/manageiq/pull/21215 so less than 5.2.6 has issues with mimemagic I believe.  I can open it up to just 5.2